Source/controls/plrctrls.cpp

```diff
@@ -1509,6 +1509,9 @@ void UseBeltItem(int type)
 {
 	for (int i = 0; i < MAXBELTITEMS; i++) {
 		auto &myPlayer = Players[MyPlayerId];
+		if (myPlayer.SpdList[i].isEmpty()) {

```

Comment:
get rid of the same check below?

Reply:
Thanks, I also made it somewhat readable, hope you agree :)